src/logic/operate.js

```diff
@@ -0,0 +1,18 @@
+import Big from 'big.js';
+
+const operate = (nOne, nTwo, operation) => {
+  let total = null;
+  const n1 = Big(nOne);
+  const n2 = Big(nTwo);
+  if (operation === '-') total = n1 - n2;
+  else if (operation === '+') total = n1 + n2;
+  else if (operation === 'X') total = n1 * n2;
+  else if (operation === '÷') {
+    if (n2 === '0') total = 'Cant divide by 0';
+    else total = n1 / n2;
+  }
+
```

Comment:
- [ ] Kindly replace the operators here with big.js math operators. Eg: `total = n1.plus(n2)`.